manifests/init.pp

```diff
@@ -183,9 +134,33 @@
     '/usr/share/containers/oci/hooks.d': ;
   }
 
+  if $manage_cilium {
+    if fact('os.name') == 'Ubuntu' and fact('os.release.full') == '22.04' {
+      # prepare system for cilium
+      # see https://docs.cilium.io/en/v1.13/operations/system_requirements/#systemd-based-distributions
+      ini_setting { 'ManageForeignRoutes':
```

Comment:
If possible, I would prefer if the Systemd module could handle these.

Reply:
saddly,  it is not possible to set these values with the systemd module atm.